client/homebrew/brewRenderer/tutorialPopup/tutorial.jsx

```diff
@@ -0,0 +1,161 @@
+                        </span>
+                    </div>
+                    <div className="brew">
+                        <span className="explanation brew">
+                            This is the Brew pane, it displays a preview of the
+                            document
+                        </span>
+                    </div>
+                </div>
+            );
+        },
+        snippetsStep: function () {
+            return (
+                <div className="content">
+                    <i className="fas fa-arrow-left"></i>
+                    <span className="explanation">
+                        This is the Snippet Bar, here you will find pieces of
+                        code to help you write
+                    </span>
+                    <span>
+                        Be sure to check on this every now and again, new
+                        snippets come every once in a while
+                    </span>
+                </div>
+            );
+        },
+        tabsStep: function () {
+            return (
+                <div className="content">
+                    <span className="explanation">
+                        These are the three tabs, Brew, Style and Properties
+                    </span>
+                    <span className="explanation">
+                        You can use each one to modify the text and structure,
+                        style and properties of the document, respecively
+                    </span>
+                </div>
+            );
+        },
+        navigationStep: function () {
+            return (
+                <div className="content">
+                    <i className="fas fa-arrow-right"></i>
+                    <span className="explanation">
+                        This is the navigation bar, here you will find different
+                        options to help you navigate and find what you are
+                        looking for
+                    </span>
+                </div>
+            );
+        },
+    },
+
+    checkTutorial: function () {
+        const hideDismiss = localStorage.getItem(DISMISS_KEY);
+        if (hideDismiss) return this.setState({ steps: {} });
+
+        this.setState({
+            steps: _.mapValues(this.steps, (fn) => fn()),
+        });
+    },
+
+    dismiss: function () {
+        localStorage.setItem(DISMISS_KEY, true);
+        this.checkTutorial();
+    },
+
+    getNextStep: function () {
+        const steps = Object.keys(this.steps);
+        const currentStepIndex = steps.indexOf(this.state.currentStep);
+        const nextStepIndex = currentStepIndex + 1;
+
+        if (nextStepIndex < steps.length) {
+            const nextStep = steps[nextStepIndex];
+            this.setState({ currentStep: nextStep });
+        } else {
+            // If we've reached the end of the steps, dismiss the tutorial
+            this.dismiss();
+        }
+    },
+
+    renderControls: function () {
```

Comment:
Placing the buttons within the actual explainer divs would be more intuitive to me-- the explainers should already be drawing the eye to themselves so it makes sense to place the related buttons right there with them.  In particular, if something is annoying me on a screen like a popup, I want the dismiss button to be within the immediate vicinity of that annoyance, not some other faraway place on the screen.